### ISSUE TITLE
Nuget CLI

### ISSUE THREAD

**Issue by NathanTheGr8** [NuGet/NuGetGallery#3294]
How do you add nuget to path environment variable?
I don't know how to do that and all your [official](https://docs.nuget.org/ndocs/guides/install-nuget#nuget-cli) install guide for nuget cli says is 
"If desired, add that location to your PATH environment variable so you can NuGet from anywhere. "
Not very detailed instuctions. A link to how to do that or more detailed instructions in the guide would be greatly appreciated.

I have found this [thread](http://superuser.com/questions/444109/regarding-the-way-to-add-a-directory-to-the-path-environment-under-windows), but adding a new one name "nuget" with a value of "C:\Program Files\Nuget\nuget.exe" (random place I decided to put .exe) didn't allow me to just type nuget "some command". 

So in short how do I edit the windows path environment variable so that I can just type "nuget" and not "\path\to\nuget.exe" Thank you for your help.

**Comment by wpostma** [comment]
I hope you'll learn that this WAS me being friendly. I answered your question and suggested how you can learn to collaborate with people more effectively in future.

**Comment by yishaigalatzer** [comment]
Hey Nathan,

Thanks for the feedback, we generally try to keep our docs reasonably which is why we didn't go into this detail.

@karann-msft will take a look at how we can update the doc without being too superfluous.

For your particular issue, I think you just have to reopen the command prompt/powershell console after updating the path variable.

**Comment by shishirx34** [comment]
Elaborated in [StackOverflow](http://stackoverflow.com/questions/4822400/register-an-exe-so-you-can-run-it-from-any-command-line-in-windows)
```
You have to put your .exe file's path into enviroment variable path. Go to "My computer -> properties -> advanced -> environment variables -> Path" and edit path by adding .exe's directory into path.
```
Note to put the __directory__ in the PATH and not .exe itself.

**Comment by NathanTheGr8** [comment]
@yishaigalatzer I did try relaunching cmd/powershell after editing path variable and even a restart to see if it only took effect on boot.

@shishirx34 I edited my path to be just the directory and I still can't get it to work. Here is a [screen shot](http://imgur.com/cJ5wkEX) of my Window. I am assuming I am missing something simple. 

Also is there a convention or typical place I am supposed to store nuget.exe?

**Comment by wpostma** [comment]
Did you look at the imgur link I pasted to you? Look at it please:

https://i.stack.imgur.com/JphSc.png

The variable you modify should be already there and should be named PATH

It consists of folder names, NOT EXE FILE NAMES, and they are separated by semicolons.

On Linux you might use PATH=/usr/bin:/usr/sbin on Windows paths are usually like this:

PATH=c:\windows\system32;c:\windows;c:\Program Files\nugetdir

If you put nuget.exe into directory c:\Program Files\nugetdir you add c:\Program Files\nugetdir to your path.

**Comment by NathanTheGr8** [comment]
ah, I did look at the link you had posted earlier but fail read it thoroughly. I didn't want to create a system variable at first so I thought I would just need to put it under my user ones. Thanks for your help.

**Comment by wpostma** [comment]
It's okay to CREATE a user-variable PATH and it's OK to modify (append) something to the system path. It depends on YOU, and what you want.

When you log in as a different user, your user-specific values won't be there. That's why I told you it matters whether you want this to work for YOU, or for a build system, or whatever.

**Comment by NathanTheGr8** [comment]
aww so I just need to make one under user named "path" with a value of "\path\to\binaryfolder\" like [this](http://imgur.com/eJKNSSU)

**Comment by wpostma** [comment]
Yep. I think people said that like three times, and the Stackoverflow link and imgur link actually gave you a PICTURE.  have a great day.

**Comment by yishaigalatzer** [comment]
@karann-msft please make a decision if you want to take a change to the docs, and resolve the bug accordingly.

**Comment by karann-msft** [comment]
I'll try and find the official documentation on adding/modifying environment variables and link to it from our docs. However, we won't be adding steps to do that explicitly in our docs.
